src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/SendPolicyRequirement.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Bit.Core.AdminConsole.Enums;
+using Bit.Core.AdminConsole.Models.Data.Organizations.Policies;
+using Bit.Core.Enums;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements;
+
+/// <summary>
+/// Policy requirements for the Disable Send and Send Options policies.
+/// </summary>
+public class SendPolicyRequirement : IPolicyRequirement
+{
+    public bool DisableSend { get; init; }
+    public bool DisableHideEmail { get; init; }
+
+    public static SendPolicyRequirement Create(IEnumerable<PolicyDetails> policyDetails)
+    {
+        var filteredPolicies = policyDetails
+            .ExemptRoles([OrganizationUserType.Owner, OrganizationUserType.Admin])
+            .ExemptStatus([OrganizationUserStatusType.Invited, OrganizationUserStatusType.Revoked])
+            .ExemptProviders()
+            .ToList();
```

Comment:
❓ How does this affect the filtering already performed by the client? There's a very similar function in the service, IIRC, that I needed to update. Is that being migrated to the server?

Reply:
Great question! Today, client and server each have their own separate-but-identical implementations of this logic. This PR refactors the server implementation but doesn't change any functionality - so no corresponding client changes are required at this point in time.

However, in the future I intend to include these PolicyRequirement objects in sync data, so that clients can rely on the server to just give them the answer without having to reimplement this logic. That would make the client-side `PolicyService` basically just a wrapper around state management and nothing else.